app/routes/abstract-module-route.js

```diff
@@ -1,6 +1,8 @@
-import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
 import Ember from 'ember';
+import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
```

Comment:
I find it more pleasing to the eye when i have ember on top, of course not a good reason. do you not like it there ? @taras do you have an opinion ?